index.bs

```diff
@@ -115,8 +115,100 @@ The <dfn for="a" element-attr>attributionreportto</dfn> attribute optionally dec
 The <dfn for="a" element-attr>attributionexpiry</dfn> attribute optionally defines the amount
 of time in milliseconds the attribution source should be considered for reporting.
 
-Issue: Need monkey patches passing attribution source in navigation, and a mechanism
-for validating the resulting document matches the attributiondestination.
+<h3 id="monkeypatch-navigation">Navigation</h3>
+
+This section ensures that an [=attribution source=] associated with a navigation
+results in a top-level navigation whose final url is [=same site=]
+[=attribution source/attribution destination=].
+
+<h4 id="monkeypatch-navigation-params">Navigation Params</h4>
+
+A <a spec="HTML">navigation params</a> struct has an item:
+
+: <dfn for="navigation params">attribution source</dfn>
+:: null or an [=attribution source=] declared when initiating a navigation
+
+<h4 id="monkeypatch-navigate-algorithm">Navigate algorithm</h4>
+
+Modify the <a spec="html">navigate</a> algorithm to accept a new optional parameter
+<var>attributionSource</var> of type [=attribution source=] defaulting to null.
+
+In <a spec="html">navigate</a>, within step
+
+> 19. This is the step that attempts to obtain resource, if necessary. Jump to the first appropriate substep:
+>     ...
+
+in the case where
```

Comment:
Can you write something about why it's correct to drop the attribution source for the other cases?

Reply:
Done. Upon looking at this further I don't think there is a good reason to not handle javascript schemes, so I added that and a note on the other case.